katas/mumbling/README.md

```diff
@@ -0,0 +1,20 @@
+# Mumbling Kata
+
+The goal of this kata is to implement the `accum()` method which takes a string as input and returns a formatted output string.
+
+The following examples illustrate the `accum()` method:
```

Comment:
Don't like `accum` - would suggest we come up with a better method name on our own repo

Reply:
have amended it to mumble_letters - I'm not sure exactly what to call it, is this fine?